contracts/socket/andromeda-socket-astroport/src/contract.rs

```diff
@@ -396,22 +347,47 @@ fn provide_liquidity(
                     expires: None,
                 };
                 let allowance_wasm_msg = wasm_execute(contract_addr, &allowance_msg, vec![])?;
-                response = response.add_message(allowance_wasm_msg);
+                response_msg.push(allowance_wasm_msg);
             }
         }
     }
 
     // Send the provide liquidity message to the pair (native coins attached, CW20s via allowance)
     let provide_wasm_msg = wasm_execute(pair_addr_raw, &provide_liquidity_msg, native_coins)?;
-    response = response.add_message(provide_wasm_msg);
+    response_msg.push(provide_wasm_msg);
+
+    let mut response: Response = Response::new();
+    for msg in response_msg {
```

Comment:
There's no need for this loop since we can directly do `response.add_messages(response_msg);`